### ISSUE TITLE
Add PIL

### ISSUE THREAD

**Issue by thipokKub** [rationalmatter/Juno-Issue-Tracker#190]
Try to read jpg image using `matplotlib.pylot.imread` and got the following error thrown

```ModuleNotFoundError                       Traceback (most recent call last)
/var/mobile/Containers/Data/Application/E3958279-DF32-4201-828F-8F251B187543/Library/Application Support/com.rationalmatter.junoapp/python-home/lib/python3.6/site-packages/matplotlib/image.py in imread(fname, format)
   1411         try:
-> 1412             from PIL import Image
   1413         except ImportError:

ModuleNotFoundError: No module named 'PIL'
```

**Comment by navoshta** [comment]
Pillow (PIL fork) will be available in the next TestFlight build for Juno 1.3.1 (currently in review) — if anyone would like to help test it, please reach out to beta@juno.sh, and I'll add you to our beta list. 😉

**Comment by navoshta** [comment]
Juno 1.3 with Pillow is now live on App Store 🚀
